netadapter/netconnection.go

```diff
@@ -33,3 +33,9 @@ func (c *NetConnection) ID() *id.ID {
 func (c *NetConnection) Address() string {
 	return c.connection.Address().String()
 }
+
+// SetOnBadMessageHandler sets a handler function
+// for bad messages
+func (c *NetConnection) SetOnBadMessageHandler(onBadMessageHandler server.OnBadMessageHandler) {
```

Comment:
Messgae is bad... mkay?

But seriously, be a bit more specific as to what a "bad" message is? Is it malformed? Invalid? Unparseable?